### PR TITLE
Frontend(auth): Add email verification feature on client side

### PR DESCRIPTION
**Pull request by spyshiv**
Now email verification link can be verified on client side.
Additional change: code quality improved in services.js

@deshraj : Please review

### CODE REVIEW

File: frontend/src/js/controllers/authCtrl.js [hunk 1]
```diff
@@ -194,6 +194,28 @@
...
+            vm.startLoader("Verifying Your Email");
+            var parameters = {};
+            parameters.url = 'auth/registration/account-confirm-email/' + $state.params.email_conf_key + '/';
+            parameters.method = 'GET';
+            parameters.callback = {
+                onSuccess: function(response) {
+                    vm.email_verify_msg = "Your email has been verified successfully";
+                    vm.stopLoader();
+                    $timeout(function() {
+                        $state.go("auth.login");
```
Comment (deshraj) [inline frontend/src/js/controllers/authCtrl.js]: lets not redirect the user directly to the login page. Lets give a button on that view from where the user can go to Login View.

Comment (spyshiv) [inline frontend/src/js/controllers/authCtrl.js]: okk. cool

**Comment by spyshiv** [comment]
@deshraj : Please review again.

File: frontend/src/js/services/services.js [hunk 2]
```diff
@@ -32,41 +32,17 @@
...
-                    method: parameters.method,
-                    url: url,
-                    headers: headers
-                };
+            var req = {
+                method: method,
+                url: url,
             }
-
+            if (header == 'header') {
```
Comment (aka-jain) [inline frontend/src/js/services/services.js]: @spyshiv : don't we need get and delete method too?

Comment (spyshiv) [inline frontend/src/js/services/services.js]: See line 35-37 . Default will work for GET and DELETE method.

Comment (aka-jain) [inline frontend/src/js/services/services.js]: Got it, nice modification. Thanks :)

Comment (spyshiv) [inline frontend/src/js/services/services.js]: :sweat_smile:

**Pull request merged**